tools/purge_non_existing_images.sh

```diff
@@ -1,29 +1,52 @@
 #!/bin/sh
 
+DO=no
```

Comment:
In general i think it does indeed make sense.

One small remark, `DO` variable name sounds kinda generic.
Personally i'd use something somewhat resembling `dry-run`, from `git clean`.
(And yes, the default should be to **NOT** do anything.)

I did not test the changes though.